corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/views.js

```diff
@@ -1,12 +1,21 @@
 'use strict';
-/*global Marionette */
-
-hqDefine("cloudcare/js/formplayer/apps/views", function () {
-    var constants = hqImport("cloudcare/js/formplayer/constants"),
-        FormplayerFrontend = hqImport("cloudcare/js/formplayer/app"),
-        googleAnalytics = hqImport("analytix/js/google"),
-        kissmetrics = hqImport("analytix/js/kissmetrix");
-
+hqDefine("cloudcare/js/formplayer/apps/views", [
+    'jquery',
+    'underscore',
```

Comment:
I don't see underscore on the left side in 417b7c238c099552a9e60f77334a97b3362f061e.

Reply:
`hqImport` is only for HQ js modules (check out its [definition](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/static/hqwebapp/js/hqModules.js#L101) - in the pre-requirejs world, all HQ js modules are stored in a big global). Third party libraries typically define globals that we then access directly. Usually you'll see them in a `globals` comment at the top of the file, but the common ones like underscore don't need that (due to [this](https://github.com/dimagi/commcare-hq/blob/master/.eslintrc.js#L19) eslint config). In the requirejs world, we're expecting all dependencies, both HQ and third party, to be explicitly declared.